### DISCUSSION TITLE
Vielen Dank für dieses Projekt !!!

### DISCUSSION THREAD

**Discussion by karlo62** [matlen67/soyosource-powercontroller#4] [General]
Ich bin schon so einige Zeit mit diesem Projekt beschäftigt, und möchte mich dafür bedanken. Leider habe ich es nicht so mit dem Programmieren und so haben meine Bemühungen noch nicht so richtig Erfolg gebracht. Ich habe mit Arduino gearbeitet.
Zuerst wurde mein Shelly 3em als Shelly em erkannt und dann gar nicht mehr.
Ich habe eben mal die fertige BIN geflasht, und sieh da ...... es läuft. Erkannt als Shelly 3em und alle 3 Phasen werden richtig angezeigt.
Ich muss mich wohl nochmal damit beschäftigen.
Am Mittwoch werde ich es an meinen 3 Soyosource testen. Vor allem das ein Teiler und ein Offset vorhanden ist .... Genial !!!!!

**Comment by matlen67** [comment]
Hey moin, ja da waren noch ein paar Bugs vorhanden, da ich selber nur einen 3EM Pro habe und das Auslesen halt anders als beim 3EM funktioniert. Das freunt mich das es jetzt geht. Viel Spaß --Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 29.04.24, 20:03 schrieb karlo62 ***@***.***>:

   
  Ich bin schon so einige Zeit mit diesem Projekt beschäftigt, und möchte mich dafür bedanken. Leider habe ich es nicht so mit dem Programmieren und so haben meine Bemühungen noch nicht so richtig Erfolg gebracht. Ich habe mit Arduino gearbeitet. Zuerst wurde mein Shelly 3em als Shelly em erkannt und dann gar nicht mehr. Ich habe eben mal die fertige BIN geflasht, und sieh da ...... es läuft. Erkannt als Shelly 3em und alle 3 Phasen werden richtig angezeigt. Ich muss mich wohl nochmal damit beschäftigen. Am Mittwoch werde ich es an meinen 3 Soyosource testen. Vor allem das ein Teiler und ein Offset vorhanden ist .... Genial !!!!!

**Comment by karlo62** [comment]
Ich konnte es nicht abwarten und habe den ESP8266 mal ausgetauscht, und jetzt läuft das Nulleinspeiseprojekt matlen67 als Datenbereitsteller für meine 3 Soyos. 
Sieht bis jetz Top aus, der Teiler funktioniert und auch der Offset. Was mir auch gut gefällt ist das die Summe der erzeugten Leistung beim Output Soyo angezeigt wird. Tip Top sage ich.
Sehr gute Arbeit !!!

**Comment by Den176** [comment]
vielen Dank für deine Arbeit, hab die bin Datei für ESP8266 auf einen D1 mini pro geflasht und siehe da, es funktioniert :-)
Leider hab ich von Programmierung keine Ahnung aber ich benötige einen größeren Teiler.
Habe 4 Soyo 1000 dran hängen. Wäre es möglich diesen Teiler im Programm auf bis zu 5 zu erhöhen?

**Reply by matlen67** [reply]
moin, habe den Teiler auf 6 erhöht
